js/src/ui/DappCard/dappCard.js

```diff
@@ -65,34 +71,29 @@ export default class DappCard extends Component {
         />
         <div className={ styles.description }>
           <ContainerTitle
-            className={ styles.title }
-            title={
+            className={
               showLink
-                ? this.renderLink(app)
-                : app.name
+                ? styles.titleLink
+                : styles.title
             }
+            title={ app.name }
             byline={ app.description }
           />
-          <div className={ styles.author }>
-            { app.author }, v{ app.version }
-          </div>
           { children }
         </div>
       </Container>
     );
   }
 
-  renderLink (app) {
-    return (
-      <Link
-        to={
-          app.url === 'web'
-            ? '/web'
-            : `/app/${app.id}`
-        }
-      >
-        { app.name }
-      </Link>
-    );
+  getLink (app) {
+    const { showLink } = this.props;
+
+    if (!showLink) {
+      return null;
+    }
+
+    return app.url === 'web'
+      ? '/web'
+      : `/app/${app.id}`;
```

Comment:
Can't we have this piece of logic somewhere else where it could be accessible from every component? I can imagine that it could be useful to have an easy access to this function.